app/src/main/res/values/color_palette.xml

```diff
@@ -203,4 +205,29 @@
   <color name="color_palette_onboarding_4_status_bar_color">@color/color_def_onboarding_4_status_bar</color>
   <color name="color_palette_coming_soon_rect_background_end_color">@color/color_def_grape_violet</color>
   <color name="color_palette_coming_soon_rect_background_start_color">@color/color_def_bright_violet</color>
+  <!-- AVATAR BACKGROUND COLORS -->
+  <color name="color_palette__avatar_background_1_color">@color/color_def_avatar_background_1</color>
```

Comment:
Nit: there's an extra underscore in the middle of all of these new color names--please fix.

Reply:
Done.